src/common/SPELLS/rogue.ts

```diff
@@ -578,6 +578,14 @@ const spells = spellIndexableList({
     name: 'Audacity',
     icon: 'ability_rogue_ambush',
   },
+
+  //Tiers
+
+  OUTLAW_ROGUE_TIER_28_2P_SET_BONUS: {
```

Comment:
I would recommend renaming this to `OUTLAW_ROGUE_T28_2P_PROC` or something similar rather than `_BONUS`, but that's nitpicky and not for this PR.